### ISSUE TITLE
About dataset processing issues?

### ISSUE THREAD

**Issue by huhuhud** [LiQiufu/WaveCNet#7]
Hello, It's very nice of you to share this code. I am very sorry because I can't find any code about dataset processing.I dowmload the ILSVRC2012 dataset and unzip the "ILSVRC2012_img_train.tar", "ILSVRC2012_img_val.tar", "ILSVRC2012_img_val.tar", respectively. How can I do next? And What is the data format of a in your folder“/raid/liqiufu/DATA/ILSVRC2012/train” and “/raid/liqiufu/DATA/ILSVRC2012/val”?  Where do I enter the label of the picture in the code? Thank you very much.

**Comment by LiQiufu** [comment]
ImageNet1000 is a standard dataset. You can easily process it in PyTorch. Please read code in https://pytorch.org/vision/stable/_modules/torchvision/datasets/imagenet.html#ImageNet
